GUI/LinuxUtil.h

```diff
@@ -0,0 +1,47 @@
+#ifndef _LINUXUTIL_
```

Comment:
Just a heads up, you're mixing both the old and the new format in this and the cpp file. Use anything from the `System` folder as reference to the new format.
No need to fix this right now but please do before you submit the next PR, otherwise it's just another thing that will need reformatting.